instrumentation/opentelemetry-instrumentation-click/tests/test_click.py

```diff
@@ -162,6 +168,27 @@ def command_raises():
             },
         )
 
+    def test_uvicorn_cli_command_ignored(self):
+        @click.command("uvicorn")
+        def command_uvicorn():
+            pass
+
+        runner = CliRunner()
+        result = runner.invoke(command_uvicorn)
```

Comment:
I think we could just call `from uvicorn.main import main as uvicorn_command` and use it instead of mocking it.

It seems uvicorn is mocked, but Flask is not.

---

Please do not ask me to create an issue. I'm good. 🙏

Reply:
If you look at the code flask is recognized by checking a parameter type since the command name `run` is pretty generic. uvicorn name is pretty unique..